internal/test_cases/receive_replication_handshake_test_case.go

```diff
@@ -62,17 +63,56 @@ func (t ReceiveReplicationHandshakeTestCase) RunReplconfStep1(client *resp_conne
 }
 
 func (t ReceiveReplicationHandshakeTestCase) RunReplconfStep2(client *resp_connection.RespConnection, logger *logger.Logger) error {
-	commandTest := ReceiveAndSendTestCase{
-		Assertion: resp_assertions.NewWildcardCommandAssertion("REPLCONF", "capa", "*", "?capa", "*"),
+	commandTest := ReceiveCommandTestCase{
+		Assertion: resp_assertions.NewOnlyCommandAssertion("REPLCONF"),
 		Response:  resp_value.NewSimpleStringValue("OK"),
 	}
 
-	return commandTest.Run(client, logger)
+	err := commandTest.Run(client, logger)
+	if err != nil {
+		return err
+	}
+
+	receivedValue := commandTest.ReceivedValue
+
+	if receivedValue.Type != resp_value.ARRAY {
+		return fmt.Errorf("Expected array type, got %s", receivedValue.Type)
+	}
+
+	elements := receivedValue.Array()
+
+	if len(elements) < 3 {
+		return fmt.Errorf("Expected array with at least 3 element, got %d elements", len(elements))
+	}
+
+	firstCapaArg := elements[1].String()
+
+	if elements[1].Type != resp_value.SIMPLE_STRING && elements[1].Type != resp_value.BULK_STRING {
+		return fmt.Errorf("Expected replconf argument to be a string, got %s", elements[1].Type)
```

Comment:
```suggestion
		return fmt.Errorf("Expected first replconf argument to be a string, got %s", elements[1].Type)
```

(Makes it more clear what you're testing)